man/systemd.network.xml

```diff
@@ -1381,6 +1381,17 @@
             service type to CS6 (network control) or CS4 (Realtime). Defaults to CS6.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>MultiPathRoute=<replaceable>address</replaceable> [dev=<replaceable>name</replaceable>] [weight=<replaceable>number</replaceable>]</varname></term>
```

Comment:
i think we should try to avoid "double keying" settings, i.e. have additional config keys inside of config key values. Can't we fine a different way to configure this? 

at the least, maybe we can do away with the key names, i.e. instead write it like this:

MultiPathRoute=address@iface weight

i.e. do something like we did for IO bandwidth settings in unit files, where you write:

IODeviceBandwidth=/dev/sda 5M

I mean, the simple ini file concept we generally use is very simplistic, and it isn't really set up to deal with complex objects, but I'd always take this as encouragement to find way to "flatten out" these objects psychologically and hide the fact that some things are multidimensional, by pretending they are uni-dimensional... Hence, using explicit "subfield" names make clear these are multidimensional, but if we instead just have pairs of space separated values this doesn't become as obvious, if you follow at all what I am babbling here...